### DISCUSSION TITLE
Display time on OLED in yyyy-MM-dd hh:mm (24 hour) format

### DISCUSSION THREAD

**Discussion by Slyke** [arendst/Tasmota#19987] [Q&A]
Hello, trying to figure out how to display a the time in specific format.

I checked here: https://tasmota.github.io/docs/Displays/#displaytext

But it seems the date and time format is only DD.MM.YY and HH.MM with [Tt].

Was hoping something a little closer to ISO8601 or RFC 3339.

**Comment by sfromis** [comment]
Closest is if you instead have a rule displaying `%timestamp%`, which includes seconds and optionally time zone.
